app/pb_kits/playbook/pb_date_picker/sass_partials/_day_styles.scss

```diff
@@ -0,0 +1,47 @@
+// Day and Range Styles
+.flatpickr-day {
+  user-select: none;
+  @media (hover: hover) {
+    &:hover {
+      transition-duration: $transition_short;
+      transition-timing-function: $easeIn;    
+      background-color: #F6FAFF;
```

Comment:
Can this be a variable?

Reply:
Had originally used variable noted in the mocks, `$neutral` I think.  Last round of styling feedback ask to match colors on mock so I grabbed the hex values manually.  Mock color value was not `$neutral`.  Can change back to `$neutral` if need be.